README.md

```diff
@@ -1,58 +1,91 @@
-- Collaboration and Feedback: Code reviews provide a platform for constructive feedback, discussions, and ideas that can lead to better solutions and continuous improvement.
-- Codebase Health: Code reviews can help us prevent technical debt and help identify both risky low-quality code, and areas for improvement in our developmebt process and tooling.
-- Onboarding and Knowledge Transfer: Reviews help us onboard new team members by exposing them to our codebases and practices.
+- __Maintain code quality and correctness:__ Code reviews help identify and address logic errors, bugs, code style departures, and security vulnerabilities. It ensures that code being merged meets our quality standards and helps us to evolve those standards.
+- __Knowledge sharing:__ Reviews are the main forum for discussions of real code and should facilitate elevated conversations across the team. PR reviews provide an opportunity for engineers to learn from one another, share best practices, and gain insights into the different parts of the codebase.
+- __Consistency:__ Reviews help ensure that the whole team follows the same conventions and coding standards, leading to a more readable and maintainable codebase.
+- __Collaboration and feedback:__ Code reviews provide a platform for constructive feedback, discussions, and ideas that can lead to better solutions and continuous improvement.
+- __Codebase health:__ Code reviews can help us prevent technical debt, identify risky code patterns, and discover areas for improvement in our development process and tooling.
+- __Onboarding and knowledge transfer:__ Reviews help us onboard new team members by exposing them to our codebases and practices.
 
 ## PR Expectations
 
-### PR Description
-
-- PR naming conventions
-- Title
-- Link to ticket
-- What's Here: Describe what you did and why, before and after
-- Steps to test
-- Questions/What I'd Like Feedback On
-- Other References
-
 ### Code Expectations
 
-- PRs should be small both so that reviews are digestible and to reduce deployment risks
-- Code should conform to linting/style expectations
-- All functions/methods should have comment blocks in the standard documentation format for the language (generally docblock)
-- Unit test coverage is expected and reported
-- Commits follow semantic commits
+- PRs should be small. Meaningful reviews require digestible changes. Small changes also reduce deployment risks.
+- Code should conform to linting/style expectations.
+- All functions/methods should have comment blocks in the standard documentation format for the language (generally docblock).
+- Ideally, all code should be checked in alongside tests that validate its
+  correctness. These should include unit tests, at a minimum. Larger features
+  should include integration tests (possibly in parallel PRs, depending on
+  feature size).
+- Use semantic commit messages.
```

Comment:
Ok, can we include the glossary of terms we use here if we're promoting this?

Reply:
Yes -- do y'all have a recommended link? I'm not sure what your policies are? I'm just going to add my suggestions as a stab for commits, but we can/should link to standards if we're going to use them.